src/index.tsx

```diff
@@ -193,15 +195,21 @@ const waitForDocumentInfo = async () => {
 }
 waitForDocumentInfo();
 
+const hither: HitherInterface = {
+  createHitherJob: createHitherJob
+}
+
 const content = (
   // <React.StrictMode> // there's an annoying error when strict mode is enabled. See for example: https://github.com/styled-components/styled-components/issues/2154 
-  <MuiThemeProvider theme={theme}>
-    <Provider store={theStore}>
-      <Router>
-        <AppContainer />
-      </Router>
-    </Provider>
-  </MuiThemeProvider>
+  <HitherContext.Provider value={hither}>
```

Comment:
And so here's where the actual hither context being used gets inserted, right?

Reply:
That's right.